src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
+}
+
+export const UserDetailsForm: React.FunctionComponent = ({status,setStatus}: any) => {
+
+  const [userDetails, setUserDetails] = useState<User>({
+    name: null,
+    displayPic:null,
+  });
+  
+
+  function onSubmit(event: any) {    
+    event.preventDefault();     
+    setStatus(true);   
+  }
+
+  return (
+    <div>
+      <h2>Enter your details cadet !</h2>
+   
+      <form action="post">
+        <div className="name">
+          <label> Name: </label>
+          <input
+            name="Name"
+            type="text"
+            onChange={(e) =>
+              setUserDetails({ ...userDetails, name: e.target.value })
+            }
+          />
+        </div>
+
+        <div className="displayPic">
```

Comment:
Class names should use `kebab-case` not `camelCase`:
```suggestion
        <div className="display-pic">
```
But as above, I don't think you should be giving this a class name! I'm not even sure it should be inside a `<div>`!